cli/pcluster/commands.py

```diff
@@ -685,6 +704,23 @@ def delete(args):
         sys.exit(0)
 
 
+def delete(args):
+    PclusterConfig.init_aws(config_file=args.config_file)
+    LOGGER.info("Deleting: %s", args.cluster_name)
+    stack_name = utils.get_stack_name(args.cluster_name)
+    stack_exists = utils.stack_exists(stack_name)
+    if args.keep_logs and not stack_exists:
```

Comment:
I'd include this case in the `not stack_exists` case. I mean the command is to delete the cluster, if it's not there, we can avoid to check the `keep_logs` argument.

Reply:
Sounds good. This dovetails with what fdm said in one of his comments as well.

Reply:
Done.